content/_data/changelogs/lts.yml

```diff
@@ -5732,6 +5726,378 @@
     message: |-
       Improve build status progress animation.
 
+- version: "2.302.1"
+  date: 2021-08-25
+  lts_predecessor: "2.289.3"
+  lts_baseline: "2.302"
+  changes: # compared to lts_baseline 2.302 - extracted from the RC commit(s)
+
+  - type: bug
+    category: bug
+    pull: 5613
+    issue: 66094
+    authors:
+    - trungPa
+    message: |-
+      Fix an issue unzipping archives in a corner case when entries have the same path prefix as the target location.
+
+  - type: bug
+    category: bug
+    pull: 5634
+    issue: 65923
+    authors:
+    - benebsiny
+    message: |-
+      Show tooltips when users hover on the SVG icons.
+
+  - type: rfe
+    category: rfe
+    pull: 5628
+    issue: 66177
+    authors:
+    - amuniz
+    message: |-
+      Ability to disable Java 11 administrative monitor with a system property.
+
+  lts_changes: # compared to lts_predecessor 2.289.3 (selected by personal review)
+
+  - type: major bug
```

Comment:
Doesn’t major bug come before bug?

Reply:
> Doesn’t major bug come before bug?

It does, but in this case there are two sections in the changelog.  The first section is used to highlight backported fixes.  We highlight backported fixes because they are more likely to be interesting to users, since they were deemed important enough to be specifically pulled into the LTS.  The second section is the changes since the previous LTS (2.289.3).  This major bug is the first item in the "changes since the previous LTS" section